components/property_info.rst

```diff
@@ -0,0 +1,295 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertInfo
+
+The PropertyInfo Component
+==========================
+
+    The PropertyInfo component extracts information about PHP class' properties
```

Comment:
I would say it's "properties of the PHP class", but I'm not sure (I remember having the same discussion with "the classes constructor").